### PR TITLE
Added function for variable default count of logs loaded.

### PR DESCRIPTION
**Pull request by mjnovice**
This adds a function to evaluate the number of logs that should be loaded, depending on the height of the viewport.

### CODE REVIEW

**Comment by rtnpro** [comment]
@mjnovice Always add a description to a PR :)

File: waartaa/client/subscribe.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ ChatSubscribe = function () {
   Default initial chat log count to load for each room.
   A room may be a channel, server or PM room.
   */
-  DEFAULT_LOGS_COUNT = 30;
+  DEFAULT_LOGS_COUNT = Math.floor(($(window).height()/100)*5);
```
Comment (rtnpro) [inline waartaa/client/subscribe.js]: @mjnovice No space around binary operators? Follow coding guidelines. You college students are pathetic at following coding guidelines :P

Comment (mjnovice) [inline waartaa/client/subscribe.js]: Fixing it rightaway!

**Comment by rtnpro** [comment]
This issue is not so simple as it seems. I will need to do some research to find the best possible way to fix this.

**Pull request closed without merging**